.doc_gen/metadata/auto-scaling_metadata.yaml

```diff
@@ -162,15 +162,16 @@ auto-scaling_DeleteAutoScalingGroup:
         - sdk_version: 3
           github: python/example_code/auto-scaling
           excerpts:
-            - description: Update the minimum size of an Auto Scaling group to zero, terminate all
+            - description:
```

Comment:
Why update Python?

Reply:
Looks like prettier kicked in and reformatted to a column width. Reverted.